### ISSUE TITLE
Implement `xml-behavior` feature in metaschema

### ISSUE THREAD

**Issue by wendellpiez** [usnistgov/OSCAL#468]
# User Story:

We have designed but not yet built out a feature to permit groupings of fields within assemblies to be made explicit in the XML. By default, a metaschema permits a sequence of fields or assemblies to be given within their parent assembly, with the metaschema available to disambiguate ordering among siblings. For XML data this should be the usual case.

However when `xml-behavior='GROUPED`` is given, on the XML side a wrapping element should be introduced.

So, for
```xml
<model>
  <field name="prop" min-occurs='unbounded'>
    <group-as name="properties" xml-behavior='GROUPED'/>
  </field>
</model>
```
instead of permitting

```xml
<prop>one</prop>
<prop>two</prop>
```
this should specify

```xml
<properties>
  <prop>one</prop>
  <prop>two</prop>
</properties>
```
i.e. a grouping element is given explicitly.

## Goals:

{A clear and concise description of what you want to happen. This should be outcome focused. Include concise description of any alternative solutions or features you've considered. Feel free to include screenshots or examples about the feature request here.}

## Dependencies:

Note that this impacts:

- XSD generation
- XML to JSON conversion
- JSON to XML converion

## Acceptance Criteria

- [ ] The XML behavior functionality `GROUPED` can be demonstrated in demo and/or actual metaschemas. " (`UNGROUPED` is the default and the way things are working currently.)
- [ ] All [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes in this issue have been updated. Changes to the OSCAL website can be made in the docs/content directory of your branch.
- [ ] A Pull Request (PR) is submitted that fully addresses the goals of this User Story. This issue is referenced in the PR.
- [ ] The CI-CD build process runs without any reported errors on the PR. This can be confirmed by reviewing that all checks have passed in the PR.

{The items above are general acceptance criteria for all User Stories. Please describe anything else that must be completed for this issue to be considered resolved.}

**Comment by david-waltermire-nist** [comment]
@wendellpiez We should be sure that this doesn't drive unintended changes to the XML schema for our existing models. We want to avoid changes that are unnecessary.

**Comment by wendellpiez** [comment]
Rolling into this Issue:

1. We have to implement not only `group-as/@in-xml` (`GROUPED`|`UNGROUPED`) but also `field[@as-type='markup-multiline']/@in-xml` (`UNWRAPPED`|`WITH-WRAPPER`)
2. Additionally, build out not only the schemas and Schematron and converters but also
  * Schema documentation
  * Schema model maps

This last may require some design work.

In PR #484 we currently have the implementation of `@in-xml` on markup multiline fields. Working on group-as behavior.

**Comment by iMichaela** [comment]
## 9/5/2019 ##
Issue was implemented and is part of PR #484 and needs to be tested (we need to develop unit tests)

**Comment by david-waltermire-nist** [comment]
This was completed in PR #492.
